.idea/allianceauth.iml

```diff
@@ -6,21 +6,16 @@
         <option name="rootFolder" value="$MODULE_DIR$" />
         <option name="settingsModule" value="settings.py" />
         <option name="manageScript" value="manage.py" />
+        <option name="environment" value="&lt;map/&gt;" />
       </configuration>
     </facet>
   </component>
   <component name="NewModuleRootManager">
     <content url="file://$MODULE_DIR$" />
-    <orderEntry type="inheritedJdk" />
+    <orderEntry type="jdk" jdkName="Python 2.7.6 (/usr/bin/python2.7)" jdkType="Python SDK" />

```

Comment:
> PyCharm is an Integrated Development Environment (IDE) used for programming in Python.

And I'm just learning about it *NOW?!* Here I am coding in the console on my VM...